<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="4">
     <widget class="QLCDNumber" name="lcdMakshastighed">
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
    color: rgb(255, 0, 0);    
    background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Afstand til forhindring m</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLCDNumber" name="lcdHastighed">
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
    color: rgb(0, 255, 0);    
    background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Acceleration G</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLCDNumber" name="lcdAcceleration">
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
    color: rgb(0, 255, 0);    
    background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLCDNumber" name="lcdAfstand">
      <property name="styleSheet">
       <string notr="true">QLCDNumber{
    color: rgb(0, 255, 0);    
    background-color: rgb(255, 255, 255);
}</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="OpretForbindelse">
      <property name="text">
       <string>Opret forbindelse</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Makshastighed km/h</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Hastighed km/h</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="KonfigurerIP">
      <property name="text">
       <string>Konfigurer IP</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="AKS">
      <property name="text">
       <string>AKS</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="KalibrerStyretoj">
      <property name="text">
       <string>Kalibrer styret√∏j</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="LukNed">
      <property name="text">
       <string>Luk ned</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="8" colspan="3">
     <widget class="VlcWidgetVideo" name="video" native="true"/>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="IndstilMaksHastighed">
      <property name="text">
       <string>Indstil Makshastighed</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header>vlc-qt/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>OpretForbindelse</tabstop>
  <tabstop>KonfigurerIP</tabstop>
  <tabstop>AKS</tabstop>
  <tabstop>KalibrerStyretoj</tabstop>
  <tabstop>LukNed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
